sections/appendix/cat_auth_pachinko.tex

```diff
@@ -19,8 +19,16 @@ \subsection{Category and Author Pachinko}\label{app:cat_auth_pachinko}
 		Taxonomy \acrlong{pam} & 0.660 \\
 		\textbf{Author \acrlong{pam}} & \textbf{0.598} \\
 		\textbf{Category \acrlong{pam}} & \textbf{0.585} \\
+		\textbf{\acrlong{pam}} & textbf{0.670}
 	\end{tabular}
 	\label{tab:cat_auth_pachinko_results}
 \end{table}
 
-\langballe[inline]{paragraph about comparison to normal pachinko with no metadata and no locking (90-90 layers)}
+For comparison we also ran a four level \gls{pam} without any metadata, using 100 and 90 as the sizes of the two middle layers.
+This ended up providing very good results, slightly better than the Taxonomy \gls{pam}.
+This also shows that while Category \gls{pam} and Author \gls{pam} get better results than their \gls{lda} counterparts, it is better to run \gls{pam} without modifications.
+This could be due to our application, but it might also point towards these two particular metadata fields not being particularly useful in this specific application.
+The category metadata are generally very vague and some categories have seemingly no connection between documents.
+And as we have discussed earlier author metadata might not be as usefull within journalism as authors don't write about the same topics as much as with scientific papers.
+And while the \gls{pam} without metadata does achieve better topic coherence than the Taxonomy \gls{pam}, they are close enough in both topic coherence and manual inspection of the quality of topics that no conclusions can be drawn.
+It could be that Taxonomy \gls{pam} only needed more training time as it is a five level model which increases the training time and complexity significantly compared to the four level model.
```

Comment:
```suggestion
It could be that Taxonomy \gls{pam} only needed more training time as it is a five-level model which increases the training time and complexity significantly compared to the four-level model.
```

Reply:
```suggestion
It could be that the Taxonomy \gls{pam} only needed more training time as it is a five-level model, which increases the training time and complexity significantly compared to the four-level model.
```